include/ultra64/controller.h

```diff
@@ -97,7 +97,7 @@
 #define BTN_A           0x8000
 
 typedef struct {
-    /* 0x00 */ u32 ram[15];
+    /* 0x00 */ uintptr_t ram[15];
```

Comment:
This isn't a `uintptr_t`. PIF RAM is exactly 64/0x40 bytes large. Boosting that size in 64-bit mode would break horribly. afaik pointers are never stored in PIF RAM so it should never be a problem either, it's mostly used for sending commands to joybus devices (see structs in controller.h with header in the name for examples of the kinds of command structures that are passed through the PIF RAM)